### DISCUSSION TITLE
white sreen on 8048S050C

### DISCUSSION THREAD

**Discussion by h3lgi** [rzeldent/esp32-smartdisplay#119] [Q&A] [answered]
I got [this](https://a.aliexpress.com/_mqZYmAm) board

I followed all of the steps in the instructions and tried to tweak some of the settings and calls but no luck. 
The only thing that can be reached is a white screen. 

Can someone point me to where I should start the debugging process?
Also if there is a way to check if the panel is just damaged or just malfunctioned?

p.s. basic things are working:
- changing backlight
- logging touch position

**Comment by rzeldent** [comment]
Hi Oleg,

I don't know this board. Just copied the settings from the schematic I found online.
Problem right now is that boards flood the market and do not always conform to the schematic/specifications.
If you have the schematic of this board you can try to verify the board parameters... Or maybe another controller?

**Reply by h3lgi** [reply]
Unfortunately, I do not have the schematics, I already asked the seller but no response for now.

Can you share a link to the original board? 

About the controller I have no idea how to check that out, probably I can unstick the LCD panel itself and try to see if there are some identifications.

**Comment by rzeldent** [comment]
https://github.com/rzeldent/platformio-espressif32-sunton/README.md

In the assets directory are the schematics etc...

**Comment by rzeldent** [comment, marked as answer]
Think you might have a strange/wrong board. esp32-4850S050 is wortking fine.
https://github.com/rzeldent/platformio-espressif32-sunton/issues/35

**Reply by h3lgi** [reply]
unfortunately, this is not my case so I will trace all of the connections and will check each component on the board. If will find anything useful will post it somewhere. For now the only difference that I found the the version, mine is 1.2.
